### ISSUE TITLE
[bug]: The icons search detail panel is not reachable with the keyboard

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#5257]
### 🐞 Describe the bug

On the [Icons search documentation](https://next.design-system.post.ch/?path=/docs/0dcfe3c0-bfc0-4107-b43b-7e9d825b805f--docs), when clicking on a icon to see its details, the panel is not accessible using the keyboard.

### 🛠️ To Reproduce

1. Navigate to the Icons search page
2. Using the keyboard, click on one of the icons to open its details
3. Using the keyboard again, try and tab to reach the panel: the focus is moving to the next icon instead of going into the details panel, and it will only reach the panel once the last icon has been focused

### ✨ Expected behavior

Once the icon has been clicked, the focus should go to the opened panel
